WooCommerce/WooCommerceUITests/Tests/LoginTests.swift

```diff
@@ -8,7 +8,7 @@ final class LoginTests: XCTestCase {
 
         // UI tests must launch the application that they test.
         let app = XCUIApplication()
-        app.launchArguments = ["logout-at-launch", "disable-animations", "mocked-wpcom-api"]
+        app.launchArguments = ["logout-at-launch", "disable-animations", "mocked-wpcom-api", "-ui_testing"]
```

Comment:
Do you think it's important to be consistent in how we name the arguments? It looks like we don't prefix with a dash in all our other arguments. And we use dashes instead of underscores. 

```suggestion
        app.launchArguments = ["logout-at-launch", "disable-animations", "mocked-wpcom-api", "ui-testing"]
```

Reply:
I prefer naming consistency, but in this case I wanted to be consistent with [the argument we already have in `SnapshotHelper`](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/WooCommerceScreenshots/Utils/SnapshotHelper.swift#L127). That's used by the Screenshots test runs and the helper is shared with at least WordPress, so I didn't want to change the argument used there in case it gets overwritten later.